packages/webcert/src/index.css

```diff
@@ -161,4 +161,11 @@ img {
 
 .ic-link-list--nav .ic-button--secondary {
   color: #01A5A3 !important;
+}
+
+.iu-link-divider-left:before {
```

Comment:
Perhaps these styles should reside together with the links in `WebcertHeader` (and not be tied to the css framework class)?